sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
 # Release History
 
-## 4.3.0-beta.1 (Unreleased)
+## 4.3.0 (2021-06-08)
 
+### New Features
+* Added the following new system events:
+  - FarmBeatsApplicationDataChangedEventData
```

Comment:
Maybe we should use the names of the events here (e.g. `Microsoft.AgFoodPlatform.<event-name>` instead of types here?)  I feel like that might read better?

Reply:
I named them according to the new model types which I think is what users would be interacting with in general.

Reply:
Works for me!  In JavaScript land I use the event names for the same reason :-)